App/Renderers/EntityRenderers/EntityRenderer.cs

```diff
@@ -0,0 +1,30 @@
+﻿using AntEngine.Entities;
+using Microsoft.Xna.Framework;
+using Microsoft.Xna.Framework.Graphics;
+using Color = Microsoft.Xna.Framework.Color;
+
+namespace App.Renderers.EntityRenderers
+{
+    /// <summary>
+    /// Represents the renderer for an abstract Entity class.
+    /// </summary>
+    public class EntityRenderer : IRenderer
+    {
+        public static Texture2D entityCharset; 
```

Comment:
What is this ?

Reply:
The texture of the renderer! Since it must be preloaded in the main loop, this was the fastest way to get a reference to it. This is very ugly, I know!

Reply:
An entity renderer could get a reference of a Texture2D in its constructor

Reply:
True but in the case that an entity is being rendered in a canvas of a canvas of a canvas, we'd have to pass the reference to all textures all the way through, which seems annoying. But there is probably not a better way to do it